BE/src/main/java/team18/airbnb/domain/DiscountPolicy.java

```diff
@@ -8,4 +8,8 @@ public enum DiscountPolicy {
     WEEK(4.0), MONTH(8.0), YEAR(16.0);
 
     private final double percent;
+
+    public double getDiscountPolicy() {
+        return percent;
+    }
```

Comment:
```suggestion
    public double getDiscountRate() {
        return discountRate;
    }
```

함수명과 필드명을 위처럼 바꾸는게 어떨지 제안드려봅니다!

Reply:
넵 제안 감사합니다.
제안해 주시것이 더 직관적인거 같네요 ㅎ.ㅎ